config/notifies.go

```diff
@@ -15,62 +15,50 @@ package config
 
 import (
 	"fmt"
+	"strings"
 )
 
 var (
 	// DefaultHipchatConfig defines default values for Hipchat configurations.
 	DefaultHipchatConfig = HipchatConfig{
-		ColorFiring:   "purple",
-		ColorResolved: "green",
+		Color:         `{{ if eq .Status "firing" }}purple{{ else }}green{{ end }}`,
```

Comment:
I'm okay with kicking this one down to the default.tmpl. For things this brief I was thinking it may make sense to do it inline.

There's a few other equivalents that can be replaced by an if in templates, mostly color related.